implementations/rust/ockam/ockam_app/src/app/process.rs

```diff
@@ -1,8 +1,8 @@
 use crate::app::events::system_tray_on_update;
-use tauri::{AppHandle, RunEvent, Wry};
+use tauri::{AppHandle, RunEvent, Runtime};
 
 /// This is the function dispatching application events
-pub fn process_application_event(app: &AppHandle<Wry>, event: RunEvent) {
+pub fn process_application_event<R: Runtime>(app: &AppHandle<R>, event: RunEvent) {
```

Comment:
Since tauri has multiple runtimes (I think it's desktop and mobile) by using the trait bound we can be neutral, and I just think it should be the cleanest way to do it to avoid surprises on upgrade.

Reply:
>  (I think it's desktop and mobile)

That's the piece I was missing, thanks!